packages/katello/rubygem-pulp_ansible_client/rubygem-pulp_ansible_client.spec

```diff
@@ -2,7 +2,7 @@
 %global gem_name pulp_ansible_client
 
 Name: rubygem-%{gem_name}
-Version: 0.18.0
+Version: 0.18.1
```

Comment:
Probably :D

Reply:
Um, looks like 0.18.1 is not already merged to rpm/develop.  there is a pr open for 0.18.2, but since 0.18.1 is current in pulp-packaging rpm/3.28, i'm confident in keeping pulp_ansible_client as is.  pulp_deb_client did have another PR opened, so I'm removing that change from this PR

Reply:
pr to nightly opened by github actions for 0.18.2 has been merged.  https://github.com/theforeman/foreman-packaging/pull/9844